File: En Python - Interpretado/NumerosPrimos.py
#!/usr/bin/env python

numero = int(input("Ingrese un numero a evaluar: "))

cont = 0

print()
print("{0} es divisible por".format(numero), end=": ")
for n in range(1, numero+1):
    if numero % n == 0:
        print(n, end=" - ")
        cont += 1
print("Fin")

if cont == 2:
    print("El numero ingresado si es primo, tiene {0} divisores".format(cont))
else:
    print("El numero ingresado no es primo, tiene {0} divisores".format(cont))

File: En Kotlin - JVM/NumerosPrimos.kt
/*
    Autor:  Fixed Torres
    Descp:  Evaluar si un numero es primo o no
 */
package numeroprimo

fun main(arg: Array<String>) {
    var NumPrimo: Int?
    var Cont = 0
    var Lim = 0

    print("Por favor, ingrese el numero a evaluar: ")
    NumPrimo = readLine()?.toIntOrNull()

    println()
    println("Numero ingresado:\t$NumPrimo")
    print("Es divisible por:\t")
    (1 .. NumPrimo!!).forEach { div ->
        if(NumPrimo % div == 0) {
            print("$div ")
            Cont++
            Lim++
            if(Lim == 10) {
                print("\n\t\t\t")
                Lim = 0
            }
        }
    }
    println("Fin!!")

    when(Cont) {
        2 -> println("Si es un numero primo, tiene $Cont divisores")
        else -> println("No es un numero primo, tiene $Cont divisores")
    }
}
